app/controllers/stories_controller.rb

```diff
@@ -14,6 +14,8 @@ class StoriesController < ApplicationController
   before_action :authenticate_user!, except: %i[index search show]
   before_action :set_cache_control_headers, only: %i[index search show]
 
+  rescue_from ArgumentError, with: :bad_request
```

Comment:
Love that we are able to keep this contained to this single controller like this! I'm betting we find more uses for this statement in a couple of other controllers as well 😉